ReactWindows/ReactNative.Net46/ReactNative.Net46.csproj

```diff
@@ -4,18 +4,19 @@
   <PropertyGroup>
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
     <Platform Condition=" '$(Platform)' == '' ">x86</Platform>
-    <ProjectGuid>{98EFA5D4-A185-4971-AE77-3845E31B84CE}</ProjectGuid>
+    <ProjectGuid>{22CBFF9C-FE36-43E8-A246-266C7635E662}</ProjectGuid>
     <OutputType>Library</OutputType>
     <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>ReactNative.Shared.Tests</RootNamespace>
-    <AssemblyName>ReactNative.Shared.Tests</AssemblyName>
+    <RootNamespace>ReactNative.Net46</RootNamespace>
+    <AssemblyName>ReactNative.Net46</AssemblyName>
     <TargetFrameworkVersion>v4.6</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|x86'">
     <PlatformTarget>x86</PlatformTarget>
     <OutputPath>bin\x86\Debug\</OutputPath>
     <DefineConstants>DEBUG</DefineConstants>
+    <DocumentationFile>bin\x86\Debug\ReactNative.Net46.XML</DocumentationFile>
   </PropertyGroup>
```

Comment:
Looks like you only enabled documentation for Debug|x86, when you check "XML Documentation File", make sure you've selected "All Configurations" and "All Platforms" from the top dropdown boxes in the settings.